src/routes/Accounts/AccountTabs/Overview.tsx

```diff
@@ -0,0 +1,231 @@
+  },
+  {
+    action: 'Sent',
+    amount: '21',
+    to: '4321abc23fdabc43ws',
+    date: '2021-05-13 14:04:45',
+    memo: 'Here’s the payment',
+  },
+  {
+    action: 'Received',
+    amount: '4',
+    to: null,
+    date: '2021-05-13 14:04:45',
+    memo: 'ty',
+  },
+  {
+    action: 'Sent',
+    amount: '104',
+    to: 'Curtis',
+    date: '2021-05-13 14:04:45',
+    memo: 'Let me know if you get this',
+  },
+  {
+    action: 'Sent',
+    amount: '21',
+    to: '4321abc23fdabc43ws',
+    date: '2021-05-13 14:04:45',
+    memo: 'Here’s the payment',
+  },
+  {
+    action: 'Received',
+    amount: '4',
+    to: null,
+    date: '2021-05-13 14:04:45',
+    memo: 'ty',
+  },
+  {
+    action: 'Sent',
+    amount: '104',
+    to: 'Curtis',
+    date: '2021-05-13 14:04:45',
+    memo: 'Let me know if you get this',
+  },
+]
+
+const AccountOverview: FC<AccountOverviewProps> = props => {
+  return (
+    <>
+      <Flex w="100%" pb="2rem">
+        <Box
+          layerStyle="card"
+          // p="2rem"
+          bg="linear-gradient(92.65deg, #85ADFE 0.41%, #4D88FF 100.03%) !important"
+          borderRadius="0.25rem"
+          w="100%"
+          minWidth="18rem"
+          mr="1rem"
+        >
+          <Flex justifyContent="space-between" alignItems="center">
+            <Box m="2rem">
+              <Box>
+                <chakra.h4>$IRON</chakra.h4>
+              </Box>
+              <Box mb="0.5rem">
+                <chakra.h2>8,456,435.4563</chakra.h2>
+              </Box>
+              <Box>
+                <Button
+                  variant="primary"
+                  borderRadius="4rem"
+                  mr="1rem"
+                  borderColor="transparent"
+                  leftIcon={
+                    <Icon height={8}>
+                      <Send fill="currentColor" />
+                    </Icon>
+                  }
+                >
+                  <h5>Send</h5>
+                </Button>
+                <Button
+                  variant="primary"
+                  borderRadius="4rem"
+                  borderColor="transparent"
+                  mr="1rem"
+                  leftIcon={
+                    <Icon height={8}>
+                      <Receive fill="currentColor" />
+                    </Icon>
+                  }
+                >
+                  <h5>Receive</h5>
+                </Button>
+              </Box>
+            </Box>
+            <Box display={{ base: 'none', md: 'inline-block' }} m="1rem">
+              <AccountKeysImage />
+            </Box>
+          </Flex>
+        </Box>
+        <Box
+          layerStyle="card"
+          p="2rem"
+          borderRadius="0.25rem"
+          minWidth="17.5rem"
+        >
+          <Box>
+            <chakra.h4>Pending $IRON</chakra.h4>
+          </Box>
+          <Box mb="0.5rem">
+            <chakra.h2>143.456</chakra.h2>
+          </Box>
+        </Box>
+      </Flex>
+      <CommonTable
```

Comment:
Minor for now but Transactions and search are missing
![image](https://user-images.githubusercontent.com/39185158/182854418-9a2acf0a-7b4e-4682-9e55-10a4c12a3f86.png)

Reply:
Fixed.